### PR TITLE
Using custom Python Social Auth Strategy

### PR DESCRIPTION
**Pull request by clintonb**
This strategy incorporates all of the settings we previously set manually (but clearly forgot to set for this project and the cookiecutter template).

ECOM-6842

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: course_discovery/settings/local.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
 INTERNAL_IPS = ('127.0.0.1',)
 
 # AUTHENTICATION
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
```
Comment (davec-edx) [inline course_discovery/settings/local.py:38]: Do we need to do this in the devstack settings file as well?

**Review by rlucioni — APPROVED** [review]

File: course_discovery/settings/local.py (hunk 1 above)
Comment (rlucioni) [reply course_discovery/settings/local.py:38]: @davec-edx I think so, nice catch.

Comment (clintonb) [reply course_discovery/settings/local.py:38]: Done.

**Pull request merged**